src/ansible_navigator/__init__.py

```diff
@@ -1,2 +1,6 @@
 """The ansible-navigator application."""
-from ._version import __version__  # noqa: F401
+
+try:
```

Comment:
Let's not have any logic in `__init__.py` and move it to a dedicated module. Having import-time calculations should be avoided when possible, or at least hidden from the public API layer.

Reply:
There is only one scenario that I am aware of where the ImportError will get hit.

Reply:
I thought the cost of a try except was minimal, especially when it wasn't thrown.... and was being improved to be almost negligible.

Reply:
Honestly, I don't think that scenario that you mentioned earlier is realistic. Have you been able to trigger it?

Reply:
But the main motivation here is to keep the package init clean.

Reply:
If you have the deps installed and don't ever install ansible-navigator:

```
(venv) x1 ➜  src git:(build) python -m ansible_navigator --version
__main__.py source
(venv) x1 ➜  src git:(build) 
```

which has be necessary in the past when troubleshooting the share directory path resolution and manually putting it's contents in different locations vs. what setuptools would do.

Reply:
Does this mean that it is a supported mechanism — having people install the runtime deps manually but not installing/building the distribution itself?

Also, you seem to be confusing setuptools and pip. The former is a PEP 517 build back-end and the latter is a build front-end (which is what unpacks the content into the target locations).

Reply:
It mean as a owner of the repo there may be times where troubleshooting of share directory related issues you will need to run directly from source without having built or installed it. I cannot think of a better explanation than "source" as the fallback, because this is the only scenario where it could/would realistically be hit, and you would be indeed running from source.

Reply:
I am open to suggestions about what else to put here........... and understanding when else it could be hit......

Reply:
I think that if we're committed to supporting this case, there should be a test added verifying that it keeps working.

Reply:
I agree that might be a valuable test to have, although it not not within the scope of this PR.

Reply:
@webknjaz I am resolving this conversation, the use case for running from "source" is real and I've had to do it in the past to resolve share directory location issues across different operating systems and in instances when navigator is installed in system locations as root on macOS.  This PR will not include tests for that scenario.